lib/Translations/Download.php

```diff
@@ -37,19 +37,29 @@ class Download extends ServiceBase {
 	 */
 	public function download() : array {
 		$this->authenticate();
-		$downloads = [];
+		$downloads    = [];
+		$locales      = $this->config->get_locales();
+		$first_locale = current( $locales );
+
+		foreach ( $locales as $locale ) {
+
+			// Wait for 100 ms between requests, if we are not on the first request.
+			if ( $locale !== $first_locale ) {
+				usleep(100000);
```

Comment:
Move the number to a constant or property of the class. We should probably allow for control lover this value, in case it's too high/low for some scenarios.